packages/esm-patient-chart-app/src/patient-chart/action-menu/action-menu.scss

```diff
@@ -98,6 +98,18 @@ $actionPanelOffset: 3rem;
   }
 }
 
+:global(.omrs-breakpoint-lt-desktop) {
+  .hiddenSideRail {
+   visibility: hidden;
```

Comment:
Thanks @usamaidrsk, we should probally use `display:none` since the visibility property only hides the bottom navigation and preserving the layout. The aim is to use the space freed by the bottom navigation to enable tablet user have better UX